pkg/skaffold/event/event.go

```diff
@@ -677,6 +695,14 @@ func ResetStateOnBuild() {
 	handler.setState(newState)
 }
 
+// ResetStateOnTest resets the test, deploy, sync and status check state
+func ResetStateOnTest() {
```

Comment:
This needs corresponding `TestInProgress`, `TestCancelled`, `TestFailed` and `TestComplete` functions

Reply:
Removed the Test events.